modules/tile-converter/src/i3s-converter/i3s-converter.js

```diff
@@ -87,12 +87,12 @@ export default class I3SConverter {
     }
 
     const options = await this._fetchPreloadOptions();
-    const sourceTilesetJson = await load(inputUrl, CesiumIonLoader, options);
-
-    /* TODO/ib - get rid of confusing options warnings, move into options sub-object */
-    // const tilesetJson = await load(inputUrl, CesiumIonLoader, {
-    //   'cesium-ion': preloadOptions
-    // });
+    const sourceTilesetJson = await load(inputUrl, CesiumIonLoader, {
+      'cesium-ion': options['cesium-ion'],
```

Comment:
What is the problem here? Is `cesium-ion` optional but we complain if not provided?

Or should we just copy all options instead of cherry-picking?
```suggestion
     ...options,
```

Reply:
I just wanted to use options which is required only for cesium-loader to avoid header warninigs.
Fixed.